python/ray/tune/utils/placement_groups.py

```diff
@@ -170,6 +170,26 @@ def resource_dict_to_pg_factory(spec: Optional[Dict[str, float]]):
     return PlacementGroupFactory([bundle])
 
 
+def pg_factory_to_resources(
+        placement_group_factory: PlacementGroupFactory) -> Resources:
+    """Translates a PlacementGroupFactory into a Resources object.
+
+    Args:
+         placement_group_factory (PlacementGroupFactory): The
+            PlacementGroupFactory object to create a Resources object from.
+
+    Returns:
+        Resources: The Resources object.
+    """
+    counts = defaultdict(int)
+    for bundle in placement_group_factory._bundles:
+        for resource_type, count in bundle.items():
+            counts[resource_type.lower()] += count
+    num_cpus = counts.pop("cpu", 1)
+    num_gpus = counts.pop("gpu", 0)
+    return Resources(cpu=num_cpus, gpu=num_gpus, **counts)
```

Comment:
This is probably too coarse. The `cpu`/`gpu` should have the CPUs/GPUs of the first bundle, all other bundle CPUs/GPUs should be in `extra_cpu`/`extra_gpu`. `memory` and `object_store_memory` (though deprecated) should be popped as well, and remaining resources have to be passed in an `extra_resources` dict.

Reply:
done

Reply:
Are we still using this?